packages/reports/src/Containers/statement.tsx

```diff
@@ -235,47 +299,26 @@ const Statement = ({
     );
 };
 
-Statement.propTypes = {
-    action_type: PropTypes.string,
-    account_statistics: PropTypes.object,
-    component_icon: PropTypes.string,
-    currency: PropTypes.string,
-    data: MobxPropTypes.arrayOrObservableArray,
-    date_from: PropTypes.number,
-    date_to: PropTypes.number,
-    error: PropTypes.string,
-    filtered_date_range: PropTypes.object,
-    handleDateChange: PropTypes.func,
-    handleFilterChange: PropTypes.func,
-    handleScroll: PropTypes.func,
-    has_selected_date: PropTypes.bool,
-    is_empty: PropTypes.bool,
-    is_loading: PropTypes.bool,
-    is_mx_mlt: PropTypes.bool,
-    is_switching: PropTypes.bool,
-    is_virtual: PropTypes.bool,
-    onMount: PropTypes.func,
-    onUnmount: PropTypes.func,
-};
-
-export default connect(({ modules, client }) => ({
-    action_type: modules.statement.action_type,
-    account_statistics: modules.statement.account_statistics,
-    currency: client.currency,
-    data: modules.statement.data,
-    date_from: modules.statement.date_from,
-    date_to: modules.statement.date_to,
-    error: modules.statement.error,
-    filtered_date_range: modules.statement.filtered_date_range,
-    handleDateChange: modules.statement.handleDateChange,
-    handleFilterChange: modules.statement.handleFilterChange,
-    handleScroll: modules.statement.handleScroll,
-    has_selected_date: modules.statement.has_selected_date,
-    is_empty: modules.statement.is_empty,
-    is_loading: modules.statement.is_loading,
-    is_mx_mlt: client.standpoint.iom || client.standpoint.malta,
-    is_switching: client.is_switching,
-    is_virtual: client.is_virtual,
-    onMount: modules.statement.onMount,
-    onUnmount: modules.statement.onUnmount,
-}))(withRouter(Statement));
+export default withRouter(
+    connect(({ modules, client }: TStores) => ({
```

Comment:
@sergei-deriv Nice, So now you get types for `client`, But `modules.statement` is `any`.
`modules.statement` is a local store in `@deriv/reports` package, So we should extend `TStore` type in this package to add types for `StatementStore`.

I've shared an example of how to do it [here](https://github.com/mahdiyeh-deriv/deriv-app/pull/37#discussion_r1069147497), If you needed help let me know so we can do it together 🙇🏻

Reply:
> @sergei-deriv Nice, So now you get types for `client`, But `modules.statement` is `any`. `modules.statement` is a local store in `@deriv/reports` package, So we should extend `TStore` type in this package to add types for `StatementStore`.
> 
> I've shared an example of how to do it [here](https://github.com/mahdiyeh-deriv/deriv-app/pull/37#discussion_r1069147497), If you needed help let me know so we can do it together 🙇🏻

Please, look now. Is it that what you meant?